recipes/tsil/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.30.0"
+
+
+class TsilConan(ConanFile):
+    name = "tsil"
+    license = "GPL-2.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.niu.edu/spmartin/TSIL/"
+    description = "Two-loop Self-energy Integral Library"
+    topics = ("conan", "high-energy", "physics", "hep", "two-loop", "integrals")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "TSIL_SIZE": ["TSIL_SIZE_LONG", "TSIL_SIZE_DOUBLE"]
```

Comment:
Our convention is lowercase... this is completely optional

```suggestion
        "size": ["TSIL_SIZE_LONG", "TSIL_SIZE_DOUBLE"]
```

You'll need to change it in other spots

Reply:
I see your point. If it is optional and you don't mind, I'd keep it as `TSIL_SIZE`, because this is the name the authors gave to this flag. So, I would think it is less confusing to have it called `TSIL_SIZE` for a user of the package and also regarding the documentation in the original paper for TSIL.

Reply:
I aggree with @prince-chrismc, prefer lowercase options names and values, regardless of upstream names. Moreover `tsil_` is redundant with recipe name.

I would prefer something like `size: ["long", "double"]`

Reply:
Ok, I'll change it.

Reply:
Done in commit e6819ebd